source/Editor.js

```diff
@@ -1680,11 +1680,11 @@ var addLinks = function ( frag, root, self ) {
 
                 child.textContent = data.slice( index, endIndex );
                 parent.insertBefore( child, node );
+                node.data = data = data.slice( endIndex );
             }
             else {
```

Comment:
Shouldn't this parse through all the matching links? What you're talking about would exit out the first time it encountered something that wasn't a url.